### PR TITLE
Bug fix for cell phone and census track

### PR DESCRIPTION
**Pull request by upasanapattnaik-eq**
Bug fix for missing values for nrt patient's cell phone [CNDIT-1472](https://cdc-nbs.atlassian.net/browse/CNDIT-1472) and census track [CNDIT-1549](https://cdc-nbs.atlassian.net/browse/CNDIT-1549). Update patient model, tests, rework logic (@alog-enquizit).

+ Code for Patient Cell_Phone uses Entity_locator_participation's cd = 'CD' and not use_cd. 
+ Addition of census_track variable. 
+ Primary_language variable updated to point to value and not code.

### CODE REVIEW

File: person-service/src/main/java/gov/cdc/etldatapipeline/person/model/dto/persondetail/Phone.java [hunk 1]
```diff
@@ -35,7 +35,10 @@ public <T extends PersonExtendedProps> T updatePerson(T personFull) {
             } else if (useCd.equalsIgnoreCase("H")) {
                 personFull.setPhoneHome(telephoneNbr);
                 personFull.setPhoneExtHome(extensionTxt);
-            } else if (useCd.equalsIgnoreCase("CP")) {
+            }
+        }
+        if (StringUtils.hasText(cd))  {
```
Comment (alog-enquizit) [inline person-service/src/main/java/gov/cdc/etldatapipeline/person/model/dto/persondetail/Phone.java:40]: This is the important change 👍

**Review by alog-enquizit — APPROVED** [review]
lgtm 👍

File: person-service/src/main/java/gov/cdc/etldatapipeline/person/model/dto/persondetail/Phone.java (hunk 1 above)
Comment (upasanapattnaik-eq) [reply person-service/src/main/java/gov/cdc/etldatapipeline/person/model/dto/persondetail/Phone.java:40]: Thank you for the update and recs!

**Review by rshanmugam-enquizit — APPROVED** [review]
ship it!

**Pull request merged**